exercises/clock/clock.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Clock do
+  defstruct hr: 0, min: 0
```

Comment:
Would you mind fully spelling out `hour` and `minute` as in done in the new `Calendar` module?